signalbot/bot.py

```diff
@@ -413,8 +413,11 @@ async def _consume_new_item(self, name: int) -> None:
         try:
             context = Context(self, message)
             await command.handle(context)
+        except NotImplementedError as e:
+            logging.error(f"[{command.__class__.__name__}] Error: handle() method not implemented")
+            raise e
         except Exception as e:
-            logging.error(f"[{command.__class__.__name__}] Error: {e}")
+            logging.error(f"[{command.__class__.__name__}] {e.__class__} Error: {e}")
```

Comment:
To make the message nicer, can we use `e.__class__.__name__` instead? What about this format:
```
f"[{command.__class__.__name__}] {e.__class__.__name__}: {e}"
```